recipes/r-qgg/meta.yaml

```diff
@@ -0,0 +1,126 @@
+{% set version = '1.1.6' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-qgg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/qgg_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/qgg/qgg_{{ version }}.tar.gz
+  sha256: dff06a1f4684622a4e60f617758254d25210f5081d814bc143906cea8b297a19
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+#  ignore_run_exports:
+#    - r-data.table
+#    - r-corpcor
+#    - r-rcpparmadillo
+#    - r-matrix
+#    - r-rcpp
+#    - r-statmod
+#    - r-mass
+#    - r-mcmcpack
+#    - r-coda
+
+
```

Comment:
```suggestion
```
Although `conda-build` reports these as unneeded, they are definitely necessary.  The output is unfortunately a bit confusing.